### PR TITLE
Updating branding to 2.1.4xx

### PR DESCRIPTION
**Pull request by livarcocc**

### CODE REVIEW

File: build/Version.props [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <VersionMajor>2</VersionMajor>
     <VersionMinor>1</VersionMinor>
-    <VersionPatch>300</VersionPatch>
+    <VersionPatch>400</VersionPatch>
```
Comment (nguerrera) [inline build/Version.props]: This flows to the installer branding now, right?

Comment (livarcocc) [reply build/Version.props]: Yes, but I forgot to update the suffix to preview. Will push a better commit.

File: build/Version.props [hunk 2]
```diff
@@ -2,7 +2,7 @@
...
     <ReleaseSuffix Condition=" '$(ReleaseSuffix)' == '' ">preview3</ReleaseSuffix>
```
Comment (nguerrera) [inline build/Version.props]: We need to change this as well, no?

Comment (livarcocc) [reply build/Version.props]: Just commented on it above.

**Comment by livarcocc** [comment]
Fixed.

**Comment by johnbeisner** [comment]
@dotnet-bot test this please

**Pull request merged**